app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :likes, foreign_key: 'users_id'
+  has_many :posts, foreign_key: 'author_id'
+  has_many :comments, foreign_key: 'users_id'
```

Comment:
- [ ] [OPTIONAL] It would be nice to harmonise your foreign key to author_id so as to avoid confusion

Reply:
Thank you so much.